drivers/bridge/setup_ip_forwarding_test.go

```diff
@@ -17,7 +17,7 @@ func TestSetupIPForwarding(t *testing.T) {
 	}
 
 	// Set IP Forwarding
-	if err := setupIPForwarding(true); err != nil {
+	if err := setupIPForwarding(true, false); err != nil {
```

Comment:
can we enable this in the TC
I think the docker0 bridge is usually a dual ipv4+ipv6 bridge, so this will be a good test for it

Reply:
changed second parameter to true